api_spec.yaml

```diff
@@ -1328,8 +1288,50 @@ paths:
+                                                                          type: string
+                                                                      userInfoEndpointQueryParams:
+                                                                          type: object
+                                                                      userInfoEndpointHeaders:
+                                                                          type: object
+                                                                      jwksURI:
+                                                                          type: string
+                                                                      oidcDiscoveryEndpoint:
                                                                           type: string
+                                                                      userInfoMap:
+                                                                          type: object
+                                                                      requireEmail:
+                                                                          type: boolean
+                                                                      clients:
+                                                                          type: array
+                                                                          items:
+                                                                              type: object
+                                                                              properties:
+                                                                                  clientType:
+                                                                                      type: string
+                                                                                  clientId:
+                                                                                      type: string
+                                                                                  clientSecret:
+                                                                                      type: string
+                                                                                  scope:
+                                                                                      type: array
+                                                                                      items:
+                                                                                          type: string
+                                                                                  forcePKCE:
+                                                                                      type: boolean
+                                                                                  additionalConfig:
+                                                                                      type: object
+
```

Comment:
You need to add required parameters spec. Not all of them are required.

Reply:
Based on the types only thirdPartyId and clientId are required. Marked them as required